src/domain/Progress/ProgressTracker/style.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+
+import { Variables } from '../../../common'
+import { styleForMargins } from '../../Spacers/services/margins'
+
+const StyledProgressTracker = styled.div`
```

Comment:
Should this automatically do `width: 50%` when there are less than or equal to 2 steps (and not mobile?)? Then we can enforce the design pattern here much easier

Reply:
They may have long text for 2 steps that need to be the whole width or on the other hand short test for 3 steps so I am not sure about enforcing them as we don't know the step labels.

Reply:
I think we should assume the labels will be short. I don't really trust people to implement it correctly and makes it impossible for us to ever update the design pattern.

Reply:
so how about three steps but with one label?  just full width?